src/components/team/Team.tsx

```diff
@@ -13,6 +14,8 @@ export default function Team(props: any) {
     .sort((a: any, b: any) => a.order - b.order);
 
   // const imageWidth = Math.round(getViewportWidth() / 4);
+
+  const socialIconColor = '#580394';
```

Comment:
This is the color in the sass file? Could you try to utalize what they explain in this article
https://blog.bluematador.com/posts/how-to-share-variables-between-js-and-sass/

Reply:
I tried to do this, but I get an error:  Property 'diwalaIconPurple' does not exist on type '(props: any) => Element'

Were you able to pull the color in JS somewhere else that I can see as an example?